crates/driver/src/infra/solver/dto/auction.rs

```diff
@@ -34,6 +34,7 @@ impl Auction {
         fee_handler: FeeHandler,
         solver_native_token: ManageNativeToken,
         flashloans_enabled: bool,
+        flashloan_default_lender: eth::Address,
```

Comment:
By now this constructor has quite a few arguments that will not change during the runtime of the driver.
Might be useful to have a builder component for those pieces of information. 🤔 
(follow up task if at all)